test/test.sh

```diff
@@ -7,11 +7,11 @@ function finish {
 
   psql -d postgres -c "drop database if exists lb_test1;"
   psql -d postgres -c "drop database if exists lb_test2;"
-}
```

Comment:
I'd create separate PR for that

Reply:
Hm....good catch, I didn't intend to change anything in the test.